Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/viewmodel/CWAViewModelSource.kt

```diff
@@ -0,0 +1,49 @@
+package de.rki.coronawarnapp.util.viewmodel
+
+import android.os.Bundle
+import androidx.lifecycle.AbstractSavedStateViewModelFactory
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.savedstate.SavedStateRegistryOwner
+import com.squareup.inject.assisted.Assisted
+import com.squareup.inject.assisted.AssistedInject
+
+class CWAViewModelSource @AssistedInject constructor(
```

Comment:
I think the implementation should be called factory as well, even though it is our source we should still stick to pattern, especially when customizing already generic code.

Reply:
So `CWAViewModelFactory.Factory` ?